angular/_darkTheme.scss

```diff
@@ -409,11 +409,11 @@
     /* PXB Empty State */
     .pxb-empty-state {
         .pxb-empty-state-empty-icon-wrapper {
-            color: map-get($pxb-black, 200);
+            color: map-get($foreground, secondary-text);
         }
```

Comment:
can we remove this rule entirely since it's the same as defined in the shared component theme?